irohad/main/application.cpp

```diff
@@ -117,9 +118,21 @@ void Irohad::initStorage() {
         throw std::runtime_error(error.error);
       });
 
+  PostgresOrderingServicePersistentState::create(pg_conn_).match(
+      [&](expected::Value<std::shared_ptr<ametsuchi::PostgresOrderingServicePersistentState>> &_storage) {
+        ordering_service_storage_ = _storage.value;
+      },
+      [](expected::Error<std::string> &error) {
+        throw std::runtime_error(error.error);
```

Comment:
wow. throwing exceptions from business logic code is forbidden without serious reasons. Better to rework it with explicit behavior.